sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/tests/QueueTriggerTests.cs

```diff
@@ -759,6 +786,16 @@ private static async Task<QueueClient> CreateQueue(QueueServiceClient client, st
             return queue;
         }
 
+        private class BindToParameterBindingData
+        {
+            public static TaskCompletionSource<ParameterBindingData> TaskSource { get; set; }
+
+            public static void Run([QueueTrigger(QueueName)] ParameterBindingData bindingData)
```

Comment:
In the similar PR for blob extension, we got feedback to also add tests with the `types` that user apps bind to. Example - `string` in case of blobs - [Reference code](https://github.com/Azure/azure-sdk-for-net/pull/31757/files#diff-6c2138179aeaa2fe65e56fcdfe9252f9bd1f475cad5903ee2acb51d02d1881e3R378). We can add similar test for Queue also if it is relevant here.

Reply:
There's already a test for binding QueueTrigger to string and other types that are already supported; the only new binding type added in this PR is ParameterBindingData so I added a test for that